consensus/hotstuff/committee.go

```diff
@@ -38,12 +38,20 @@ type Replicas interface {
 	//   * model.ErrViewForUnknownEpoch if no epoch containing the given view is known
 	LeaderForView(view uint64) (flow.Identifier, error)
 
-	// WeightThresholdForView returns the minimum weight required to form a QC
+	// QuorumThresholdForView returns the minimum total weight of votes required to form a QC
 	// for the given view. This weight threshold is computed using the total weight
```

Comment:
I feel the goDoc is a bit over specific. In addition to a QC, we also use this threshold for a Timeout Certificate. 

```suggestion
	// QuorumThresholdForView returns the minimum total weight for a supermajority 
	// at the given view. This weight threshold is computed using the total weight
```